cargo-dist/src/cli.rs

```diff
@@ -128,6 +128,9 @@ pub enum Commands {
     #[clap(disable_version_flag = true)]
     #[clap(hide = true)]
     GenerateCi(GenerateCiArgs),
+    /// Report on the dynamic libraries used by the built artifacts.
+    #[clap(disable_version_flag = true)]
+    Linkage(LinkageArgs),
```

Comment:
Ignoring the detail of human-readable output, is there any reason not to run the linkage checker internally on every run of `cargo dist build`?